man/parse_date_time.Rd

```diff
@@ -166,37 +170,37 @@ behind UTC. This format also matches the Z (Zulu) UTC indicator. Because
 strptime doesn't fully support ISO8601 this format is implemented as an
 union of 4 orders: Ou (Z), Oz (-0800), OO (-08:00) and Oo (-08). You can use
 these four orders as any other but it is rarely
-necessary. \code{parse_date_time2} and \code{fast_strptime} support all of
+necessary. \code{parse_date_time2()} and \code{fast_strptime()} support all of
 the timezone formats.}
 
 \item{\code{Om}!*}{Matches numeric month and English alphabetic months
-                   (Both, long and abbreviated forms).}
+(Both, long and abbreviated forms).}
 
 \item{\code{Op}!*}{Matches AM/PM English indicator.}
 
-\item{\code{r}*}{Matches \code{Ip} and \code{H} orders.}
-\item{\code{R}*}{Matches \code{HM} and\code{IMp} orders.}
+\item{\code{r}\emph{}{Matches \code{Ip} and \code{H} orders.}
```

Comment:
Roxygen screwed this up. It interpreted `*}` incorrectly here and as a results travis tests fail with `unknown macro \item`.  I can fix this manually for now, but I would prefer if this could be fixed upstream.